doc/Makefile

```diff
@@ -28,7 +28,5 @@ pgbouncer_1.md: filter.py frag-usage-man.md usage.md
 pgbouncer_5.md: filter.py frag-config-man.md config.md
 	$(PYTHON) $^ >$@
 
-.INTERMEDIATE: pgbouncer_1.md pgbouncer_5.md
```

Comment:
In CI for windows these files were being cleaned up before pandoc was done using them. Since windows is one of the slowest platforms to compile on, removing this seemed worth the advantage of being able to compile in parallel.